simulation/builder.go

```diff
@@ -12,6 +12,7 @@ import (
 type Builder struct {
 	parallelEngine bool
 	monitorOn      bool
+	outputFile     string
```

Comment:
Let's use outputFileName for a string. Also change the WithOutputFile to WithOutputFileName